HanoiTowers/client/hanoitowers.cpp

```diff
@@ -317,6 +317,12 @@ void HanoiTowers::handleAcceptUserData(QSharedPointer<LocalUser> data) {
 
         loadOldSaves();
 
+        if (data->gameState()->lvl() < 15) {
+            data->gameState()->saveLvl(15);
```

Comment:
* add new method for unlock level by argument into gamestate class.
* invoke this method here 
* all functions must be under defines HANOI_ADMOD and Q_OS_ANDROID 

```cpp
#ifdef Q_OS_ANDROID
#ifndef HANOI_ADMOD
// code
#endif
#endif
```